src/services/EnvironmentProvider/index.tsx

```diff
@@ -18,8 +18,8 @@ import { appendQueryParams, getQueryParam, removeQueryParam } from '../../utils'
 
 export enum Network {
   MAINNET = 'mainnet',
-  CALIBRATION = 'calibration',
-  WALLABY = 'wallaby'
+  CALIBRATION = 'calibrationnet',
+  WALLABY = 'wallabynet'
```

Comment:
I've changed these to the offical names, it makes a bit more sense to me. Like it's not `main`, but `mainnet`, so also not `calibration` but `calibrationnet`.

It also prevents us from having to fetch the official name when switching network and I don't think we really need to use `useNetworkName` in the `NetworkSwitcher` anymore, but wanted to check what you thought about this first.

For me it's fine to keep using the `useNetworkName` hook, but then I'd say we expose the official network name from the `EnvironmentProvider` as well.

Reply:
I agree we should use the official names, but the only issue is now some documentation points to the `?network=wallaby` or `?network=calibration`, so we should just handle those cases appropriately to not break backwards compatibility with those query params

Reply:
Good call, I've updated it so it accepts with or without the "net" suffix